extension/content_dashboard.js

```diff
@@ -7,16 +7,73 @@ $(function() {
     airline = AES.getAirlineCode();
     server = AES.getServerName();
     chrome.storage.local.get(['settings'], function(result) {
-        settings = result.settings;
+      settings = result.settings;
 
-        displayDashboard();
+      if ((settings['flightInformationsFixed'] === undefined) || (settings['flightInformationsFixed'] !== 1)) {
+        fixAndDeleteInvalidData();
+      }
+
+      displayDashboard();
+      dashboardHandle();
+      
+      $("#aes-select-dashboard-main").change(function() {
         dashboardHandle();
-        $("#aes-select-dashboard-main").change(function() {
-            dashboardHandle();
-        });
+      });
     });
 });
 
+function fixAndDeleteInvalidData() {
```

Comment:
Can you please adjust the indent size to .editorconfig spec?

Reply:
sure, had this set to two because the whole AES code was/is 2 spaces